torchbox/rca-wagtail-2019
# 576 Enable email notifications and configuration for study enquiries

Resolves ticket [576](https://projects.torchbox.com/projects/rca-website-rebuild/tickets/576)

![image](https://user-images.githubusercontent.com/2128707/110507427-eb6d0500-80f7-11eb-9267-9c3d4e1ea0d2.png)

### Testing:

- Go to https://rca-development.herokuapp.com/admin/settings/enquire_to_study/enquiretostudysettings/2/ to configure the email settings and save them
- Got to https://rca-development.herokuapp.com/enquire-to-study/ and fill out the form using a real email address.
- Check your email inbox and spam folders to confirm email 

This is using the EMAIL_HOST_USER as the `from` address. On the development servier this will be `generic@staging.torchbox.com`. On RCA production the from address will be `postmaster@rca.ac.uk` so that may need changing.

After submitting the form you should receive an email to the address filled out in the forms 'email' field. Please be sure to check your spam.
